pkg/defaults/provider.go

```diff
@@ -202,3 +202,12 @@ func (d *Provider) EmbeddedClusterSupportSubDir() string {
 func (d *Provider) PathToEmbeddedClusterSupportFile(name string) string {
 	return filepath.Join(d.EmbeddedClusterSupportSubDir(), name)
 }
+
+func (d *Provider) PathToHostPreflightsDir() string {
+	path := filepath.Join(d.EmbeddedClusterHomeDirectory(), "host-preflights")
```

Comment:
Wouldn't it be cleaner to store this under the `support` directory ?

Reply:
Something like `/var/lib/embedded-cluster/support/host-preflights/results.json`?

Reply:
Oh, we may want to collect this file here as well (in the case the cluster isn't working): https://github.com/replicatedhq/embedded-cluster/blob/main/pkg/goods/support/host-support-bundle.yaml